startreedata/thirdeye
# chore(ui): bump webpack-cli to get dev environment working

#### Description

Mismatches in versioning causing the following when running `npm run start`:

```
 TypeError: cli.isMultipleCompiler is not a function
    at Command.<anonymous> (/Users/spham/projects/thirdeye/thirdeye-ui/node_modules/@webpack-cli/serve/lib/index.js:146:35)
    at async Promise.all (index 1)
    at async Command.<anonymous> (/Users/spham/projects/thirdeye/thirdeye-ui/node_modules/webpack-cli/lib/webpack-cli.js:1687:7)
```

#### How to test

Fresh checkout, install node packages, then use `npm run start` to reproduce and verify fixed after
